src/types/toaster.ts

```diff
@@ -0,0 +1,12 @@
+import { AlertAppearance, AlertAppearanceRecord } from '@/types/alert'
+import { appearances } from '@/components/KAlert/KAlert.vue'
+
+export interface Toast {
+  key?: any // unique identifier of toaster
+  appearance?: AlertAppearance
+  message: string // Text to display in toaster
+  timer?: any
+  timeoutMilliseconds?: number
```

Comment:
Based on the usage [here](https://github.com/Kong/kongponents/blob/f29961b2f862926c71e9d7158e1dbf05a02e0284/src/components/KToaster/ToastManager.ts#L50-L67) shouldn't some of these properties be required? e.g. `timeoutMilliseconds` and `message`

Reply:
`message` is required.
`timeoutMilliseconds ` - I don't think it should be required, because there's a default value for it 
```
timeoutMilliseconds: timeoutMilliseconds || this.timeout,
```

Reply:
Yep let's update the message prop -- we'll need to make sure that doesn't throw errors when imported into khcp-ui